scripts/run-analyses-for-figures.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+#
+# Run all analysis modules that are used for manuscript figures
+
+#enviroment settings
+set -e
+set -o pipefail
+
+# If RUN_LOCAL is used, the time-intensive steps are skipped because they cannot
+# be run on a local computer -- the idea is that setting RUN_LOCAL=1 will allow for
+# local testing running/testing of all the other figures
+RUN_LOCAL=${RUN_LOCAL:-0}
+
+# Find current directory based on this script
+WORKDIR=$(dirname "${BASH_SOURCE[0]}")
+cd "$WORKDIR"
+
+# Get base directory of project
+cd ..
+BASEDIR="$(pwd)"
+cd -
+
+analyses_dir="$BASEDIR/analyses"
+
+#################################
+
+# Run modules that cannot be run locally due to memory requirements
+if [ "$RUN_LOCAL" -lt "1" ]; then
+  bash ${analyses_dir}/focal-cn-file-preparation/run-prepare-cn.sh       # Figures 2 and S3
+  bash ${analyses_dir}/snv-callers/run_caller_consensus_analysis-pbta.sh # Figure S2
+  bash ${analyses_dir}/snv-callers/run_caller_consensus_analysis-tcga.sh # Figure S2
+  bash ${analyses_dir}/copy_number_consensus_call/run_consensus_call.sh  # Figure S3 (heatmap)
+fi
```

Comment:
The `snv-callers` plots in S2 need the SQL db's in `scratch/`, so this is annoyingly unavoidable.

This `copy_number_consensus_call` can be removed I think, and that figure script can be updated too since results from that module don't appear to ever be read in! I might want to add in running the module `cnv-chrom-plot`, but that can be run locally outside this `if`.
https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/2a4d691498c08cc985962447806ff5f2c2b92176/figures/scripts/figS3-panels-BCD.R#L35-L39

Finally, I had kept `focal-cn-file-preparation` because of this discussion - https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/1455#issuecomment-1163114816

Reply:
> The `snv-callers` plots in S2 need the SQL db's in `scratch/`, so this is annoyingly unavoidable.

That is annoying, yes.

> Finally, I had kept `focal-cn-file-preparation` because of this discussion [#1455 (comment)](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/1455#issuecomment-1163114816), with the idea that this script serves as the "big run" after subtyping.

I think my comment there didn't take into account whether or not the output of `analyses/focal-cn-file-preparation/06-find-recurrent-calls.Rmd` _gets used_, and I don't think it does from a find all of `"consensus_seg_focal_cn_recurrent_genes.tsv"` and `"consensus_seg_recurrent_focal_cn_units.tsv"`...

Reply:
>  I don't think it does 

I agree that it doesn't. But this raises another question about how we should go about ensuring analysis modules that are in the paper, even if not directly used for figures, are up-to-date. That might be another script entirely.

Reply:
> That might be another script entirely.

Yea, that's basically where I landed.

Reply:
Oh actually we can probably scrap `snv-callers/run_caller_consensus_analysis-tcga.sh` from this script, since it's run in `generate-analysis-files-for-release.sh` and definitely doesn't need the PBTA histology file!